index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Awesome Books - Add and remove books</title>
+</head>
+
+<body>
+  <header>
+    <h1>Awesome Books</h1>
+  </header>
+  <main>
+    <div id="books-list">
+    </div>
+    <form action="#" class="add-books-form">
+      <input id="book-title" type="text" placeholder="Title" name="title" required>
+      <br><br>
+      <input id="book-author" type="text" placeholder="Author" name="author" required>
+      <br><br>
+      <input id="submitBtn" type="submit" value="Add">
+    </form>
+  </main>
+  <footer>
+    <script src="js/index.js"></script>
```

Comment:
- [ ]  OPTIONAL: I think you should put the 'script` tag just right above the body closing `</body>` tag, not inside the `footer`. Kindly make this change for your next milestone. :pray: